android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletActivity.java

```diff
@@ -104,6 +105,11 @@ public boolean onCreateOptionsMenu(Menu menu) {
         closeButtonImage.setImageResource(R.drawable.ic_baseline_close_24);
         SearchManager searchManager = (SearchManager) getSystemService(SEARCH_SERVICE);
         searchView.setSearchableInfo(searchManager.getSearchableInfo(getComponentName()));
+
+        if(menu.getClass().getSimpleName().equals("MenuBuilder")){
```

Comment:
Why don't we just check if `menu` is instance of `androidx.appcompat.view.menu.MenuBuilder` and then just cast it? Like
```
if (menu instanceof MenuBuilder) {
    ((MenuBuilder) menu).setOptionalIconsVisible(true);
}
```

Reply:
Yeah I missed there is also a `MenuBuilder` in `androidx`. Reflections are removed.